utils/tests_fetcher.py

```diff
@@ -359,8 +359,12 @@ def create_reverse_dependency_map():
     "utils/hub.py": "utils/test_hub_utils.py",
     "modelcard.py": "utils/test_model_card.py",
     "modeling_flax_utils.py": "test_modeling_flax_common.py",
-    "modeling_tf_utils.py": ["test_modeling_tf_common.py", "utils/test_modeling_tf_core.py"],
-    "modeling_utils.py": ["test_modeling_common.py", "utils/test_offline.py"],
+    "modeling_tf_utils.py": [
+        "test_modeling_tf_common.py",
+        "test_pipeline_common.py",
+        "utils/test_modeling_tf_core.py",
+    ],
+    "modeling_utils.py": ["test_modeling_common.py", "test_pipeline_common.py", "utils/test_offline.py"],
```

Comment:
Not 100% sure, but I put `test_pipeline_common.py` to both `modeling_utils.py` and `modeling_tf_utils.py`

Reply:
If it doesn't have any tests of its own (looks like it but you can confirm) and only the common tester class, you can put it in the list of test files to ignore instead, as we don't need to ever call pytest on this file.